makejdk-any-platform.1

```diff
@@ -1,11 +1,14 @@
-.\" Manpage for makejdk.sh
+.\" Manpage for makejdk-any-platform.sh
 .TH "0.1.0" "Apache 2"
 .SH NAME
-makejdk.sh
+makejdk-any-platform.sh
 .SH SYNOPSIS
-"./makejdk [options]"
+"makejdk-any-platform.sh [options]"
 .SH DESCRIPTION
-The simplest way to build OpenJDK using our scripts is to run makejdk.sh and have your user be in the Docker group on the machine (or prefix all of your Docker commands with sudo. This script can be used to create a Docker container that will be configured with all of the required dependencies and a base operating system in order to build OpenJDK.
+The simplest way to build OpenJDK using our scripts is to run makejdk-any-platform.sh and have your user be in the Docker group on the machine (or prefix all of your Docker commands with sudo. This script can be used to create a Docker container that will be configured with all of the required dependencies and a base operating system in order to build OpenJDK.
```

Comment:
Script can be used to create a Docker image ==> its a container when we run a Docker image (nit picking)